gensim/models/fasttext.py

```diff
@@ -465,29 +466,6 @@ def __init__(self, sentences=None, corpus_file=None, sg=0, hs=0, vector_size=100
             null_word=null_word, ns_exponent=ns_exponent, hashfxn=hashfxn,
             seed=seed, hs=hs, negative=negative, cbow_mean=cbow_mean, min_alpha=min_alpha)
 
-    def prepare_weights(self, update=False):
```

Comment:
These deleted methods are part of the public API. If we remove them, then shouldn't we also update documentation that refers to them?

Reply:
Aren't API docs autogenerated from doc-comments, so deleting the method/doc-comment causes the matching docs to change? 

Also, while this superficially looked like a public API (in method name and doc-comment)I don't think it was actually a good idea for anyone to call except via other more prominent methods. It's an internal step that must be called only after a bunch of specific other internal things are done. (It was only called at the end of `build_vocab()` variants, and only appropriate to call for models that had already had all the earlier vocab-establishing steps done just before.)

Reply:
Yes, the API docs are generated, but what about tutorials/notebooks that call those methods?

Reply:
Can you please grep the code for other docstrings (maybe in other modules) + notebooks that may link to these methods? I don't think any do, but it's a valid concern.

Reply:
A recursive grep of the project directory for the names of the 4 methods removed here, `reset_weights`, `update_weights`, `init_ngrams_weights`, and `update_ngrams_weights` found no external mentions of these on `FastText`. (There's still unaffected mentions, & local implementations, of `prepare_weights` on `Word2Vec` & `Doc2Vec` - but those are independent & I think I'd like to remove those in initialization-cleanup of those classes soon, too.)